purchase_discount/__manifest__.py

```diff
@@ -4,20 +4,22 @@
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
-    "name": "Purchase order lines with discounts",
+    "name": "Purchase Order Lines with Discounts",
     "author": "Tiny, "
               "Acysos S.L., "
               "Tecnativa, "
-              "ACSONE SA/NV,"
+              "ACSONE SA/NV, "
```

Comment:
There's no need for extra spaces. They are ripped out when parsed by manifest reader.